uber/templates/signups/shifts.html

```diff
@@ -133,28 +131,36 @@
             eventLimit: true, // allow "more" link when too many events
             slotEventOverlap: false,
             events: eventList,
-            eventRender: function(event,element) {
-                var buttonTag = '<a class="btn btn-primary shift_button ';
+            eventRender: function(event, element, view) {
+                var eventDesc = "";
+                if (event.description) {
+                    eventDesc += "<br/>" + event.description;
+                }
+                if (event.extra15) {
+                    eventDesc += " + 15 minute end-time."
```

Comment:
I really couldn't think of a better way to indicate that extra 15 boolean. I would like to get rid of it and just have more flexible job start and end times. :(